lib/model/product.rb

```diff
@@ -0,0 +1,58 @@
+require_relative "./../services/check_quantity"
+
+module Store
+  class Product
+    include CommonMethods
```

Comment:
not the best name, I understand what you want to do here, but CommonMethods sounds odd